test/integration/kubernetes_deploy_test.rb

```diff
@@ -534,7 +534,11 @@ def test_pruning_of_existing_managed_secrets_when_ejson_file_has_been_deleted
 
   def test_deploy_result_logging_for_mixed_result_deploy
     subset = ["bad_probe.yml", "init_crash.yml", "missing_volumes.yml", "config_map.yml"]
-    result = deploy_fixtures("invalid", subset: subset)
+    result = deploy_fixtures("invalid", subset: subset) do |f|
+      if KUBE_SERVER_VERSION >= Gem::Version.new("1.10.0")
```

Comment:
Can you please add an inline comment by the first of these changes in each test, linking to the issue you opened?